src/umpire/strategy/DynamicPool.cpp

```diff
@@ -33,17 +33,17 @@ DynamicPool::DynamicPool(const std::string& name,
                          const int align_bytes) noexcept :
   AllocationStrategy(name, id),
   m_allocator{allocator.getAllocationStrategy()},
-  m_min_alloc_bytes{min_alloc_bytes},
+  m_min_alloc_bytes{round_up(min_alloc_bytes, align_bytes)},
   m_align_bytes{align_bytes},
   m_coalesce_heuristic{coalesce_heuristic},
   m_used_map{},
   m_free_map{},
   m_curr_bytes{0},
-  m_actual_bytes{initial_alloc_bytes},
+  m_actual_bytes{round_up(initial_alloc_bytes, align_bytes)},
```

Comment:
Where do we validate `initial_alloc_bytes`? Is `initial_alloc_bytes=0` expected to work?

Reply:
Yes, I think we should UMPIRE_ERROR() if someone passes 0 as min_alloc_size, or align_bytes.

Reply:
I wrote a Jira ticket to decide what the contract needs to be for all of these parameters.  We can decide when/if/where/how to address these separately.